### ISSUE TITLE
90% das modinhas de programação é apenas uma roupa nova para algo que as pessoas já esqueceram.

### ISSUE THREAD

**Issue by xexeo** [pilhacheia/tretadev#51]

**Comment by jdoper** [comment]
Verdade, conhecer a tecnologia é super importante. Mas conceitos de arquitetura de software são eternos

**Comment by jmurowaniecki** [comment]
É…

Tipo Clipper… Que virou Recital… Que virou AdvPL.

E tipo Basic, que virou Visual Basic, que virou .NET.

E tipo C, que virou C++, que virou Java, e que terminou virando Javascript…

![](https://c.tenor.com/s2Oaghr3-ScAAAAC/michael-scott-the-office.gif)
